activestorage/app/models/active_storage/blob.rb

```diff
@@ -140,6 +141,26 @@ def combine_signed_id_purposes(purpose) #:nodoc:
     def signed_id_verifier #:nodoc:
       @signed_id_verifier ||= ActiveStorage.verifier
     end
+
+    # Interpolates the custom storage key if needed and appends to it a unique_secure_token
+    def generate_unique_interpolated_secure_key(key:, record:, attachable:, length: MINIMUM_TOKEN_LENGTH)
+      return generate_unique_secure_token(length: length) if key.blank?
```

Comment:
The purpose of this was double:
- avoid to pass through `interpolate` method uselessly
- has not the same output

If I remove this safe_guard, the key would no longer look like `secure_token_key` but `/secure_token_key` because `compact` does not remove empty String values. I can also remove this line and replace the `compact` with `delete_if(&:blank?)` on line 152.

What do you prefer?

Reply:
Ah yes, you’re right.